src/Controls/samples/Controls.Sample/Pages/Controls/ImagePage.xaml

```diff
@@ -13,24 +13,24 @@
                     Style="{StaticResource Headline}"/>
                 <Image 
                     Source="https://aka.ms/campus.jpg"
-                    HeightRequest="200" 
```

Comment:
We _can_ make requests here, but if we just leave the default, Glide will try and make the imageview bigger which seems to play well with layouts so far...